2 - RTB/Documentazione interna/Verbali/24-02-11/24-02-11.typ

```diff
@@ -0,0 +1,106 @@
+#import "/template.typ": *
+
+#show: project.with(
+  date: "11/02/24",
+  subTitle: "Meeting di retrospettiva e pianificazione",
+  docType: "verbale",
+  authors: (
+    "Oseliero Antonio",
+  ),
+  timeStart: "15:00",
+  timeEnd: "16:15",
+);
+
+= Ordine del giorno
+- Valutazione del progresso generale;
+- Analisi di retrospettiva;
+- Seconda parte RTB con #vardanega;
+- Pianificazione.
+
+== Valutazione del progresso generale
+
+Il meeting si avvia con con l'analisi dello sprint concluso. Nonostante si trattasse del primo sprint a seguito della sessione degli esami, la ripresa dei lavori è stata rapida (warm start), grazie anche alla scelta di non interrompere i lavori negli sprint precedenti ma di ridimensionare il carico di lavoro per far fronte agli impegni universitari, il corretto andamento dello sprint è supportato anche dalle metriche esposte sul cruscotto di qualità realizzato in Grafana.
```

Comment:
Cambierei in un "." tra "impegni universitari" e "il corretto". Il periodo è leggermente troppo lungo.